ocpmodels/trainers/forces_trainer.py

```diff
@@ -217,6 +219,13 @@ def train(self):
             if self.test_loader is not None:
                 self.validate(split="test", epoch=epoch)
 
+            if self.config["task"].get("ml_relax", False):
+                self.validate_relaxation(
+                        test_dir="./relax_eval",
```

Comment:
Outdated. But this will end up being the directory to the validation set. Not entirely clear how that'll look like yet to make it robust enough. For now added a line in the config to specify what directory to run these relaxations on.